configuration_go/schemas/thanos/reqlogging/reqlogging.go

```diff
@@ -0,0 +1,48 @@
+package reqlogging
+
+// Taken from https://github.com/thanos-io/thanos/blob/release-0.32/pkg/logging/yaml_parser.go
+
+type LogLevel string
+
+const (
+	LogLevelDebug LogLevel = "DEBUG"
+	LogLevelInfo  LogLevel = "INFO"
+	LogLevelWarn  LogLevel = "WARNING"
+	LogLevelError LogLevel = "ERROR"
+)
```

Comment:
These constants are almost the same as the ones in `configuration_go/schemas/thanos/common/common.go`, except that these are uppercase. Could some code be reused, maybe? Not a blocker for the PR.